backend/src/main/java/com/huellapositiva/domain/service/ImageService.java

```diff
@@ -0,0 +1,77 @@
+package com.huellapositiva.domain.service;
+
+import com.huellapositiva.application.exception.InvalidFieldException;
+import com.huellapositiva.domain.exception.EmptyFileException;
+import com.huellapositiva.domain.exception.FileTypeNotSupportedException;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.huellapositiva.domain.util.FileUtils.getExtension;
+
+@Service
+public class ImageService {
```

Comment:
:heart:

Nice idea to bring image related things to its own domain service.

Reply:
Thanks